source/common/http/null_route_impl.h

```diff
@@ -0,0 +1,242 @@
+#pragma once
+
+#include "envoy/router/router.h"
+
+#include "source/common/http/hash_policy.h"
+#include "source/common/protobuf/message_validator_impl.h"
+#include "source/common/router/config_impl.h"
+#include "source/common/upstream/retry_factory.h"
+#include "source/extensions/early_data/default_early_data_policy.h"
+
+namespace Envoy {
+namespace Http {
+
+struct NullHedgePolicy : public Router::HedgePolicy {
```

Comment:
nit: Let's add a comment at the top of this file explaining that this class is all about providing a no-op implementation of Router::Route which requires no-op implementations of a variety or Router helper classes.

Reply:
done